tdrs-backend/tdpservice/parsers/validators.py

```diff
@@ -229,6 +229,14 @@ def hasLength(length, error_func=None):
     )
 
 
+def intHasLength(num_digits):
+    """Validate the number of digits in an integer."""
+    return make_validator(
+        lambda value: len(str(value)) == num_digits,
```

Comment:
@elipe17 does this properly catch strings that include spaces or special characters? for example: `2018$$14` has only 6 digits, but it doesnt appear that we're checking the string for the number of digits in the string.